dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Trn/PreferredName.cshtml

```diff
@@ -1,7 +1,7 @@
 @page "/sign-in/trn/preferred-name"
 @model TeacherIdentity.AuthServer.Pages.SignIn.Trn.PreferredName
 @{
-    ViewBag.Title = $"Is {Model.FirstName} {Model.LastName} your preferred name?";
+    ViewBag.Title = $"Is {Model.OfficialFirstName} {Model.OfficialLastName} your preferred name?";
```

Comment:
Since it's possible for FirstName and LastName to be null here I think we should be using official names

Reply:
There's also the possibility of using preferred name here if they have previously set it. Potentially confusing when returning to this page to see us asking the question using the official name